netx/httptransport/http3transport.go

```diff
@@ -44,6 +44,9 @@ func (t *HTTP3Transport) CloseIdleConnections() {
 func NewHTTP3Transport(config Config) RoundTripper {
 	txp := &HTTP3Transport{}
 	txp.QuicConfig = &quic.Config{}
+	if config.TLSConfig != nil {
+		txp.TLSClientConfig = config.TLSConfig
+	}
```

Comment:
```suggestion
	txp.TLSClientConfig = config.TLSConfig
```

This code should work as intended if we apply the proposed changes. I like the one liner much more because it reduces the number of lines that we should be testing.

Reply:
Definitely!